djangocassandra/db/backends/cassandra/schema.py

```diff
@@ -93,7 +93,13 @@ def alter_db_table(
                (disable reads on OLD_Keyspace)
             7) clean up OLD_Keyspace (drop schema, delete files, etc.)
         '''
-        pass
+        if (
+            new_db_table._meta.db_table !=
+            old_db_table._meta.db_table
+        ):
+            raise Exception('Renaming models not supported')
+
+        self.create_model(new_db_table)
```

Comment:
Oh yeah, I also added support for the alter table which is used a lot in the migration scipts. This is the same as creating a model in cqlengine.